script.js

```diff
@@ -1,4 +1,40 @@
+var generateRandomInteger = function () {
+  var randomDecimal = Math.random() * 3;
+  var randomInteger = Math.floor(randomDecimal);
+  console.log("randomInteger = " + randomInteger);
+  if (randomInteger == 0) progInput = "Scissors";
+  if (randomInteger == 1) progInput = "Paper";
+  if (randomInteger == 2) progInput = "Stone";
+  console.log("progam input = " + progInput);
+  return progInput;
+};
+
 var main = function (input) {
-  var myOutputValue = 'hello world';
+  var progInput = generateRandomInteger();
+  // default output
+  var myOutputValue =
+    "Error, please try again. Input options are 'Scissors', 'Paper' or 'Stone'";
+  if (input == progInput)
```

Comment:
in your if statements, you could have simplified it or use comments to describe what is going on. can get hard to read and understand what you're trying to achieve